src/boardlib/__main__.py

```diff
@@ -108,6 +107,12 @@ def add_logbook_parser(subparsers):
         default="font",
         required=False,
     )
+    logbook_parser.add_argument(
+        "--db_path",
```

Comment:
Lets go with `-d` and `--database`. That way the short form is very easy to type and the long form is still clear.

Reply:
Agreed! I've made the requested changes: The argument names have been updated to "-d" and "--database".